src/test/java/bowlingexercise3/BowlingTest.java

```diff
@@ -1,34 +1,30 @@
 package bowlingexercise3;
 
-import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.hamcrest.CoreMatchers.equalTo;
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.MatcherAssert.assertThat;
 
+
 public class BowlingTest {
 
-    @Test
-    public void scoreZero() {
-        Bowling game = new Bowling();
-        game.throwws("00|00");
-        assertThat(game.score(), is(0));
-    }
 
 
-    @Test
-    public void score_7() {
-        Bowling game = new Bowling();
-        game.throwws("10|10|10|10|10|10|10|00|00|00");
-        assertThat(game.score(), is(7));
-    }
+    @ParameterizedTest
+
+    @CsvSource(value = {"--|, 0", "1-|, 1", "11|, 2", "12|, 3", "4-|, 4","32|, 5","33|, 6",
+            "61|, 7","44|, 8","45|, 9","44|2-, 10", "-/|--, 10",
+            "-/|43|-/|-/|-/|-/|45|-/|-/, 86", "-/|-/|-/|-/|-/|-/|-/|7/|-/, 90",
+            "1/|-/|4/|9/|, 40"})
```

Comment:
it would make this easier to assess if the values were grouped by what you are trying to test, rather than have everything lumped together. For example, seperate out the tests for spares into a different set of test data

Reply:
"-/|-/|-/|-/|-/|-/|-/|7/|-/, 90" - this score is incorrect, it should be 97! ("If the second ball in a frame knocks down all ten pins,  this is called a "spare". The frame is over. The score for the frame is ten plus the number of pins knocked down in the next ball.")

"-/|43|-/|-/|-/|-/|45|-/|-/, 86" - this score should be 94
"1/|-/|4/|9/|, 40" - this score should be 53

Reply:
I will now implement this scoring system and will update the code accordingly!

Reply:
> "-/|-/|-/|-/|-/|-/|-/|7/|-/, 90" - this score is incorrect, it should be 97! ("If the second ball in a frame knocks down all ten pins, this is called a "spare". The frame is over. The score for the frame is ten plus the number of pins knocked down in the next ball.")
> 
> "-/|43|-/|-/|-/|-/|45|-/|-/, 86" - this score should be 94
> "1/|-/|4/|9/|, 40" - this score should be 53

Have addressed this and corrected the method accordingly.